### PR TITLE
SAK-42396 Moved permissions webcomponent to sakai i18n

### PR DESCRIPTION
**Pull request by adrianfish**
This commit makes the permissions web component fully internationalised,
using the same translation bundle as the permissions helper in
admin-tools. Translations for admin-tools have now gone into tomcat/lib
so the Sakai i18n webservice can find them. sakai-i18n.js uses that
webservice and the permissions and rubrics components use sakai-i18n.js.

https://jira.sakaiproject.org/browse/SAK-42396

### CODE REVIEW

**Comment by adrianfish** [comment]
@mpellicer Miguel, sorry I closed the PR earlier - I noticed that the permissions wc was not even picking up the exising i18n strings for the permissions helper. I've fixed that now.

**Comment by adrianfish** [comment]
> Looking forward to having this implemented so we can use it as an archetype so all components can be i18n!

You'll just be able to extend SakaiElement, or import sakai-i18n.js and use that in your own base class.

File: commons/api/src/bundle/commons_es.properties [hunk 1]
```diff
@@ -1,6 +0,0 @@
-# this defines the entity description for the commons provider
-commons=<br />Proporciona un conjunto de puntos finales RESTful para recuperar publicaciones de COMMONS. <br /><br />
```
Comment (mpellicer) [inline commons/api/src/bundle/commons_es.properties]: What happens with the _es translation? I see that commons_es and Messages_es have been removed.

File: library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js [hunk 2]
```diff
@@ -98,6 +99,7 @@
   };
 
   Handlebars.registerHelper('tr', function (namespace, key, options) {
+      console.log("PORTAL TR");
```
Comment (mpellicer) [inline library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js]: Can we change it to debug?

File: roster2/tool/pom.xml [hunk 3]
```diff
@@ -161,7 +161,7 @@
     <build>
         <plugins>
             <!-- compress CSS and JS -->
-            <plugin>
+            <!--plugin>
```
Comment (mpellicer) [inline roster2/tool/pom.xml]: Are we losing the minify feature for the Javascript and CSS files? Well..they are not very big but is a cool feature.

File: roster2/tool/src/webapp/WEB-INF/bootstrap.jsp [hunk 4]
```diff
@@ -68,13 +68,13 @@
 
         </div> <!-- portletBody -->
 
-    <script type="text/javascript" src="/library/webjars/jquery/1.12.4/jquery.min.js"></script>
```
Comment (mpellicer) [inline roster2/tool/src/webapp/WEB-INF/bootstrap.jsp]: Well played!

File: roster2/tool/src/webapp/i18n/languages.json [hunk 5]
```diff
@@ -1,19 +0,0 @@
-{
```
Comment (mpellicer) [inline roster2/tool/src/webapp/i18n/languages.json]: well played

File: commons/api/src/bundle/commons_es.properties (hunk 1 above)
Comment (adrianfish) [reply commons/api/src/bundle/commons_es.properties]: That's no good. I'll check that.

File: library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js (hunk 2 above)
Comment (adrianfish) [reply library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js]: Thats needs to come out, tbh. Good spot!

File: roster2/tool/pom.xml (hunk 3 above)
Comment (adrianfish) [reply roster2/tool/pom.xml]: Minify broke with the current closure compiler. I'm looking at that in library at the moment. I could add that back in, now, actually. I will do.

**Pull request merged**